ova/generate_ova.sh

```diff
@@ -55,163 +64,191 @@ clean() {
 
     cd ${scriptpath}
     vagrant destroy -f
-    rm -f ${OVA_VM} ${OVF_VM} ${OVA_VMDK} ${OVA_FIXED} Config_files/filebeat.yml
-
+    OVA_VMDK="wazuh-${OVA_VERSION}-disk001.vmdk"
+    rm -f ${OVA_VM} ${OVF_VM} ${OVA_VMDK} ${OVA_FIXED}
+    
     exit ${exit_code}
 }
 
 build_ova() {
 
+    # Files
     OVA_VM="wazuh-${OVA_VERSION}.ova"
     OVF_VM="wazuh-${OVA_VERSION}.ovf"
     OVA_FIXED="wazuh-${OVA_VERSION}-fixed.ova"
-    OVA_VMDK="wazuh-${OVA_VERSION}-disk001.vmdk"
-    ELK_MAJOR=`echo ${ELK_VERSION} | cut -d"." -f1`
-    export BRANCH
-
-    if [ -e "${OVA_VM}" ] || [ -e "${OVA_VM}" ]; then
-        rm -f ${OVA_VM} ${OVF_VM}
-    fi
+    export PACKAGES_REPOSITORY
 
-    #Download filebeat.yml and enable geoip
-    if [ ${ELK_MAJOR} -eq 7 ]; then
-        curl -so Config_files/filebeat.yml https://raw.githubusercontent.com/wazuh/wazuh/v${WAZUH_VERSION}/extensions/filebeat/7.x/filebeat.yml
-        sed -i "s|#pipeline: geoip|pipeline: geoip|" Config_files/filebeat.yml
+    # Delete OVA/OVF files if exists
```

Comment:
Over-commenting

Reply:
Fixed in https://github.com/wazuh/wazuh-packages/pull/746/commits/27e2d14fcdf50546e45b5066ded7e2863482c4d4